doc/man7/openssl-quic.pod

```diff
@@ -710,6 +710,11 @@ or allow them to be handled using L<SSL_accept_stream(3)>.
 Used to configure or disable default stream mode; see the MODES OF OPERATION
 section for details.
 
+=item L<SSL_get_peer_addr(3)>
+
+Used to obtain the peer address from a QUIC connection or stream.
+
+
```

Comment:
Nit: One too many blank lines here

Reply:
Your other comment was for [doc/man3/SSL_get_peer_addr.pod](https://github.com/openssl/openssl/pull/28690/files/1d9f49c937c6edf45c89e43309ccce1701a75ef1#diff-0aa1612907f96ee14afe5bf44fbf77bbb6d96f2a2a1b03cea7d15a47f08f4c01)

I have missed this change, it comes with the next push.